charts/amazon-network-policy-controller-k8s/values.yaml

```diff
@@ -0,0 +1,50 @@
+
+replicaCount: 2
+
+image:
+  repository: public.ecr.aws/eks/amazon-network-policy-controller-k8s
+  tag: v1.0.4
+  pullPolicy: IfNotPresent
+  override: ""
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  name: ""
+
+enableConfigMapCheck: true
+endpointChunkSize: ""
+maxConcurrentReconciles: ""
+podUpdateBatchPeriodDuration: ""
+
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+
+updateStrategy: {}
```

Comment:
nit: why not define an update strategy by default?

Reply:
I didn't add it as I was trying to mimic the kustomize deployment. (Even though it was not defined, it gets a default value when it is deployed in the cluster). I'll update it to the default strategy for now